### PR TITLE
Разметка главной страницы

### PR DESCRIPTION
**Pull request by Vakarev**
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/adaptive/22/user/1610993/tasks/2)

:boom: https://htmlacademy-adaptive.github.io/1610993-pink-22/2/

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -1,12 +1,179 @@
...
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
+    <header class="page-header">
```
Comment (KirillKolomin) [inline source/index.html:8]: just `header`

File: source/index.html [hunk 2]
```diff
@@ -1,12 +1,179 @@
...
+      <div class="page-header__wrapper wrapper">
```
Comment (KirillKolomin) [inline source/index.html:9]: `page-header__wrapper` isn't needed

Comment (KirillKolomin) [inline source/index.html:9]: the `wrapper` is redundant here. move it onto the `header` tag

File: source/index.html [hunk 3]
```diff
@@ -1,12 +1,179 @@
...
+        <a class="page-header__logo">
+          <span>Pink</span>
```
Comment (KirillKolomin) [inline source/index.html:11]: should be `img`

File: source/index.html [hunk 4]
```diff
@@ -1,12 +1,179 @@
...
+        </a>
+        <nav class="main-nav">
```
Comment (KirillKolomin) [inline source/index.html:13]: just `nav`

File: source/index.html [hunk 5]
```diff
@@ -1,12 +1,179 @@
...
+          <button class="main-nav__toggle">
+            <span class="visually-hidden">Открыть меню</span>
```
Comment (KirillKolomin) [inline source/index.html:15]: is redundant

Comment (KirillKolomin) [inline source/index.html:15]: should be `img`.

Comment (KirillKolomin) [reply source/index.html:15]: with the burger icon

File: source/index.html [hunk 6]
```diff
@@ -1,12 +1,179 @@
...
+          </button>
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--active">
+              <a class="main-nav__link">Главная</a>
```
Comment (KirillKolomin) [inline source/index.html:19]: should contain the `href` attribute

File: source/index.html [hunk 7]
```diff
@@ -1,12 +1,179 @@
...
+              <a href="#" class="main-nav__link">Конкурс</a>
+            </li>
+            <li class="main-nav__item">
+              <a href="#" class="main-nav__link">Html Academy</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="page-main">
```
Comment (KirillKolomin) [inline source/index.html:34]: just main

File: source/index.html [hunk 8]
```diff
@@ -1,12 +1,179 @@
...
+      <h1 class="visually-hidden">Pink</h1>
+      <section class="promo">
+        <div class="promo__wrapper wrapper">
```
Comment (KirillKolomin) [inline source/index.html:37]: is redundant

File: source/index.html [hunk 9]
```diff
@@ -1,12 +1,179 @@
...
+          <h2 class="promo__title">Взгляните на жизнь иначе!</h2>
+          <a href="#" class="promo__link">Скачать приложение</a>
```
Comment (KirillKolomin) [inline source/index.html:39]: `promo__download` instead of

File: source/index.html [hunk 10]
```diff
@@ -1,12 +1,179 @@
...
+            <li class="features__list-item">
+              <h3 class="features__list-title">Меняет мир вокруг</h3>
+              <p class="features__list-text">Сделайте снимок и украсьте его смайликом или текстовой подписью, чтобы усилить эффект</p>
+            </li>
+            <li class="features__list-item">
+              <h3 class="features__list-title">Заводит новых друзей</h3>
+              <p class="features__list-text">Ставьте лайки, комментируйте, делитесь фотографиями с друзьями и заводите новых</p>
+            </li>
+          </ul>
+          <div class="features__item">
```
Comment (KirillKolomin) [inline source/index.html:77]: remove this leave

File: source/index.html [hunk 11]
```diff
@@ -1,12 +1,179 @@
...
+            </li>
+          </ul>
+          <p class="promo__text">Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8</p>
+        </div>
+      </section>
+      <section class="features">
+        <div class="features__wrapper wrapper">
+          <h2 class="features__title">Устали от серости мегополиса?<br>Нам есть, что вам предложить!</h2>
+          <ul class="features__list">
+            <li class="features__list-item">
```
Comment (KirillKolomin) [inline source/index.html:64]: remove the namespaces. list-item etc.

Comment (KirillKolomin) [reply source/index.html:64]: just item

File: source/index.html [hunk 12]
```diff
@@ -1,12 +1,179 @@
...
+          <ul class="reviews__list">
+            <li class="reviews__item">
+              <blockquote class="reviews__autor-info">
+                <p class="reviews__text">«Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»</p>
+                <cite class="reviews__autor-name">Николай Петров</cite>
+                <small class="reviews__autor-legend">25 лет, водитель трамвая</small>
+              </blockquote>
+            </li>
+          </ul>
+          <button class="reviews__button reviews__button--left"></button>
```
Comment (KirillKolomin) [inline source/index.html:95]: remain only mobile items

File: source/index.html [hunk 13]
```diff
@@ -1,12 +1,179 @@
...
+          <button class="reviews__button reviews__button--right"></button>
+          <ul class="reviews__toggles">
```
Comment (KirillKolomin) [inline source/index.html:97]: will be `reviews__toggles toggles`

File: source/index.html [hunk 14]
```diff
@@ -1,12 +1,179 @@
...
+            <li class="reviews__toggles-item reviews__toggles-item--active">
```
Comment (KirillKolomin) [inline source/index.html:98]: will be `toggles__item`

Comment (KirillKolomin) [inline source/index.html:98]: make only button be `--active`

File: source/index.html [hunk 15]
```diff
@@ -1,12 +1,179 @@
...
+              <button class="reviews__toggle reviews__toggle--active"></button>
```
Comment (KirillKolomin) [inline source/index.html:99]: toggle_button

File: source/index.html [hunk 16]
```diff
@@ -1,12 +1,179 @@
...
+            </li>
+            <li class="reviews__toggles-item reviews__toggles-item">
+              <button class="reviews__toggle"></button>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="price">
+        <div class="pricewrapper wrapper">
+          <table class="price__table">
```
Comment (KirillKolomin) [inline source/index.html:112]: should be a block

Comment (KirillKolomin) [reply source/index.html:112]: `price-block`

File: source/index.html [hunk 17]
```diff
@@ -1,12 +1,179 @@
...
+            <thead class="price__list-head">
+              <tr class="price__row">
+                <th class="price__value price__value--empty">
```
Comment (KirillKolomin) [inline source/index.html:115]: `price-table__cell` instead of `price-table__value`

File: source/index.html [hunk 18]
```diff
@@ -1,12 +1,179 @@
...
+                </th>
+                <th class="price__value">
+                  База
+                  <p>1,99 USD
+                </th>
+                <th class="price__value price__value--mark">
```
Comment (KirillKolomin) [inline source/index.html:121]: price-table__cell--hit instead of price__value--mark

File: source/index.html [hunk 19]
```diff
@@ -1,12 +1,179 @@
...
+                </th>
+                <th class="price__value">
+                  Анлим
+                  <p>9,99 USD
+                </th>
+              </tr>
+            </thead>
+            <tbody class="price__list-body">
+              <tr class="price__row">
+                <td class="price__field">Розовый фильтр</td>
```
Comment (KirillKolomin) [inline source/index.html:133]: make it fit for mobile layout

File: source/index.html [hunk 20]
```diff
@@ -1,12 +1,179 @@
...
+      <section class="contacts">
+        <div class="contacts__wrapper wrapper">
+          <h2 class="contacts__slogan">Остались вопросы? Свяжитесь с нами!</h2>
+          <h4 class="contacts__title">Звоните</h4>
+          <a href="tel:+78125556666">+7 (812) 555-66-66</a>
+          <h4 class="contacts__title">Пишите</h4>
+          <a href="mail@htmlacademy.ru">mail@htmlacademy.ru</a>
+          <h4 class="contacts__title">Приезжайте в гости</h4>
+          <a href="#">ул. Большая Конюшенная, 19/8</a>
+          <div class="map"><img src="img/map.png" alt="ул. Большая Конюшенная, 19/8" class="map__img"></div>
```
Comment (KirillKolomin) [inline source/index.html:163]: redundant wrapper

File: source/index.html [hunk 21]
```diff
@@ -1,12 +1,179 @@
...
+                <td class="price__field"></td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </section>
+      <section class="contacts">
+        <div class="contacts__wrapper wrapper">
+          <h2 class="contacts__slogan">Остались вопросы? Свяжитесь с нами!</h2>
+          <h4 class="contacts__title">Звоните</h4>
```
Comment (KirillKolomin) [inline source/index.html:157]: all items with h4 and a should be closed in a wrapper here.

File: source/index.html [hunk 22]
```diff
@@ -1,12 +1,179 @@
...
+          <a href="mail@htmlacademy.ru">mail@htmlacademy.ru</a>
+          <h4 class="contacts__title">Приезжайте в гости</h4>
+          <a href="#">ул. Большая Конюшенная, 19/8</a>
+          <div class="map"><img src="img/map.png" alt="ул. Большая Конюшенная, 19/8" class="map__img"></div>
+        </div>
+      </section>
+    </main>
+    <footer class="page__footer">
+      <div class="page__footer-wrapper wrapper">
+        <a class="page__footer-logo">Pink</a>
```
Comment (KirillKolomin) [inline source/index.html:169]: logos should be blocks (small, medium, large)

File: source/index.html [hunk 23]
```diff
@@ -1,12 +1,166 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
```
Comment (KirillKolomin) [inline source/index.html:2]: again. We shouldn't mark every element with class.

File: source/index.html [hunk 24]
```diff
@@ -1,12 +1,166 @@
...
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
   </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+  <body class="page__body">
```
Comment (KirillKolomin) [inline source/index.html:7]: page__body. For what?

File: source/index.html [hunk 25]
```diff
@@ -1,12 +1,166 @@
...
+              <a href="#" class="nav__link">Html Academy</a>
+            </li>
+          </ul>
+        </nav>
+    </header>
+    <main class="main">
+      <h1 class="visually-hidden">Pink</h1>
+      <section class="promo wrapper">
+          <h2 class="promo__title">Взгляните на жизнь иначе!</h2>
+          <a href="#" class="promo__download">Скачать приложение</a>
```
Comment (KirillKolomin) [inline source/index.html:36]: you have to position it on the tablet and desktop. Wrap it in a div. And you actually don't need to position it separately (promo__title and promo__download). And do the positioning of the wrapper on the mobile layout. Also notice that you have to position the ul below like both `h2` and `a` above.

File: source/index.html [hunk 26]
```diff
@@ -1,12 +1,166 @@
...
+          <ul class="platform">
+            <li class="platform__item">
```
Comment (KirillKolomin) [inline source/index.html:38]: what do you planna do with the platform__tem?

Comment (KirillKolomin) [reply source/index.html:38]: with `platform__link` too?

File: source/index.html [hunk 27]
```diff
@@ -1,12 +1,166 @@
...
+                <img src="img/android-icon.svg" width="43px" height="47px" alt="Play Market" class="platform__icon">
+              </a>
+            </li>
+            <li class="platform__item">
+              <a href="#" class="platform__link">
+                <img src="img/windows-icon.svg" width="44px" height="41px" alt="Windows Store" class="platform__icon">
+              </a>
+            </li>
+          </ul>
+          <p class="promo__text">Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8</p>
```
Comment (KirillKolomin) [inline source/index.html:54]: mb platform_description. Cuz you describe what platforms can use it on

File: source/index.html [hunk 28]
```diff
@@ -1,12 +1,166 @@
...
+      </section>
+      <section class="features wrapper">
+          <h2 class="features__title">Устали от серости мегополиса?<br>Нам есть, что вам предложить!</h2>
+          <ul class="features__list">
+            <li class="features__item">
+              <h3 class="features__list-title">Поднимает настроение</h3>
```
Comment (KirillKolomin) [inline source/index.html:60]: please don't create such name spaces `__list` ->  `__list-title`

File: source/index.html [hunk 29]
```diff
@@ -1,12 +1,166 @@
...
+                </th>
+                <th class="price-table__cell">
+                  Анлим
+                  <p>9,99 USD
+                </th>
+              </tr>
+            </thead>
+            <tbody class="price-table__body">
+              <tr class="price-table__row">
+                <td class="price__field price__field--ok">Розовый фильтр</td>
```
Comment (KirillKolomin) [inline source/index.html:117]: did you mean `price-table__cell` instead of `price__field`?

File: source/index.html [hunk 30]
```diff
@@ -1,12 +1,166 @@
...
+                <td class="price__field price__field--none">Смайлики</td>
+              </tr>
+              <tr class="price-table__row">
+                <td class="price__field price__field--ok">Комментарии</td>
+                <td class="price__field price__field--ok">Комментарии</td>
+                <td class="price__field price__field--ok">Комментарии</td>
+              </tr>
+            </tbody>
+          </table>
+          <ul class="price-table__toggles toggles">
```
Comment (KirillKolomin) [inline source/index.html:133]: are you sure it's the `price-table` (price-table__toggles)?

File: source/index.html [hunk 31]
```diff
@@ -1,12 +1,166 @@
...
+            <li class="reviews__item">
+              <blockquote class="reviews__autor-info">
+                <p class="reviews__text">«Это приложение перевернуло мой мир и позволило по-новому взглянуть на привычные серые вещи! А еще я познакомился со своей будущей женой в комментариях к выложенной фотографии!»</p>
+                <cite class="reviews__autor-name">Николай Петров</cite>
+                <small class="reviews__autor-legend">25 лет, водитель трамвая</small>
+              </blockquote>
+            </li>
+          </ul>
+          <ul class="reviews__toggles toggles">
+            <li class="toggles__item toggles__item">
```
Comment (KirillKolomin) [inline source/index.html:85]: why did you put it down twice? toggles__item toggles__item

Comment (KirillKolomin) [reply source/index.html:85]: let's do so.

toggles __|
                 |toggle 
                 |button

File: source/index.html (hunk 21 above)
Comment (KirillKolomin) [reply source/index.html:147]: still remains

File: source/index.html (hunk 23 above)
Comment (KirillKolomin) [reply source/index.html:2]: what is it for?

File: source/index.html (hunk 26 above)
Comment (Vakarev) [reply source/index.html:38]: про platform__item мы говорили, что он для margin-left нужен. А platform__link я убрал

File: source/index.html [hunk 32]
```diff
@@ -1,12 +1,174 @@
...
+    <header class="header wrapper">
+        <a class="logo-small" href="#">
+          <img class="logo-small__img" src="img/small-logo.svg" alt="Pink">
+        </a>
+        <nav class="nav">
+          <button class="nav__toggle">
+            <img class="nav__toggle-img" src="img/burger.svg" alt="Открыть меню">
+          </button>
+          <ul class="nav__list">
+            <li class="nav__item main-nav__item--active">
```
Comment (KirillKolomin) [inline source/index.html:17]: I don't see any `main-nav` blocks. `main-nav__item--active` is mistake here

File: source/index.html [hunk 33]
```diff
@@ -1,12 +1,174 @@
...
+        </nav>
+    </header>
+    <main class="main">
+      <h1 class="visually-hidden">Pink</h1>
+      <section class="promo wrapper">
+          <h2 class="promo__title">Взгляните на жизнь иначе!</h2>
+          <a href="#" class="promo__download">Скачать приложение</a>
+          <ul class="platform">
+            <li class="platform__item">
+              <a href="#">
```
Comment (KirillKolomin) [inline source/index.html:39]: forgot for links. `platform__link`?

File: source/index.html [hunk 34]
```diff
@@ -1,12 +1,174 @@
...
+              </a>
+            </li>
+          </ul>
+          <p class="platform__description">Доступно для iPhone, iPad, Android, Windows Phone, OS X, Windows 8</p>
+      </section>
+      <section class="features wrapper">
+          <h2 class="features__title">Устали от серости мегополиса?<br>Нам есть, что вам предложить!</h2>
+          <ul class="features__list">
+            <li class="features__item">
+              <h3 class="features__subtitle">Поднимает настроение</h3>
```
Comment (KirillKolomin) [inline source/index.html:60]: if it's the `features__subtitle` so where is there the `features__title`?

Comment (KirillKolomin) [reply source/index.html:60]: do you remember we decided to use `title` with `title--level-{1:6}`

File: source/index.html [hunk 35]
```diff
@@ -1,12 +1,174 @@
...
+              <p class="features__text">Приложение позволит вам победить осеннюю хандру и депрессию буквально в несколько кликов!</p>
+            </li>
+            <li class="features__subtitle">
```
Comment (KirillKolomin) [inline source/index.html:63]: Are you sure it's the `features__subtitle`?

File: source/index.html [hunk 36]
```diff
@@ -1,12 +1,174 @@
...
+              <h3 class="features__list-title">Меняет мир вокруг</h3>
```
Comment (KirillKolomin) [inline source/index.html:64]: why do you have different classes for the same meaning? `features__list-title` and `features__subtitle`

File: source/index.html (hunk 35 above)
Comment (KirillKolomin) [reply source/index.html:63]: mb it's the `features__item`?

File: source/index.html [hunk 37]
```diff
@@ -1,12 +1,174 @@
...
+                </th>
+                <th class="price-table__cell">
+                  Анлим
+                  <p>9,99 USD
+                </th>
+              </tr>
+            </thead>
+            <tbody class="price-table__body">
+              <tr class="price-table__row">
+                <td class="price-table__field price-table__field--ok">Розовый фильтр</td>
```
Comment (KirillKolomin) [inline source/index.html:117]: why for cells of this table do you have 2 classes `price-table__cell` and  `price-table__field`?

File: source/index.html [hunk 38]
```diff
@@ -1,12 +1,174 @@
...
+              </tr>
+              <tr class="price-table__row">
+                <td class="price-table__field price-table__field--ok">Комментарии</td>
+                <td class="price-table__field price-table__field--ok">Комментарии</td>
+                <td class="price-table__field price-table__field--ok">Комментарии</td>
+              </tr>
+            </tbody>
+          </table>
+          <ul class="price__toggles toggles">
+            <li class="toggles__item toggles__item">
```
Comment (KirillKolomin) [inline source/index.html:134]: don't do duplicates. `toggles__item toggles__item` ?

File: source/index.html [hunk 39]
```diff
@@ -1,12 +1,174 @@
...
+              <button class="toggles__button toggles__button--active"></button>
+            </li>
+            <li class="toggles__item toggles__item">
+              <button class="toggles__button"></button>
+            </li>
+          </ul>
+      </section>
+      <section class="contacts wrapper">
+          <h2 class="contacts__slogan">Остались вопросы? Свяжитесь с нами!</h2>
```
Comment (KirillKolomin) [inline source/index.html:146]: we decided to use `title`s for h{1:6}

Comment (KirillKolomin) [reply source/index.html:146]: with `title--level-{1:6}`

File: source/index.html (hunk 34 above)
Comment (Vakarev) [reply source/index.html:60]: 'features__title on 57

File: source/index.html (hunk 37 above)
Comment (Vakarev) [reply source/index.html:117]: price-table__cell - это ячейки-заголовки, они отличаются от prie-table__field

Comment (KirillKolomin) [reply source/index.html:117]: ячейка - есть ячейка. смысл слов cell и field здесь абсолютно одинаковы. Хочешь выделить заголовок называй  price-table__cell--header

**Pull request merged**